NuKeeper.Update/UpdateRunner.cs

```diff
@@ -64,37 +77,56 @@ private IReadOnlyCollection<PackageInProject> Sort(IReadOnlyCollection<PackageIn
         }
 
         private IReadOnlyCollection<IPackageCommand> GetUpdateCommands(
```

Comment:
If you go for factory approach you would do something like
var _dotNetUpdatePackageCommand = _dotNetUpdatePackageCommandFactory.GetDotNetUpdatePackageCommand(true/false)

then at the end add the DotNetRestoreCommand  if needed